cms/static/cms/js/modules/cms.sideframe.js

```diff
@@ -1,14 +1,29 @@
-//##############################################################################
-// #SIDEFRAME#
-/* global CMS */
-
+/*
+ * Copyright https://github.com/divio/django-cms
+ */
+
+// #############################################################################
+// NAMESPACES
+/**
+ * @module CMS
+ */
+var CMS = window.CMS || {};
+
+// #############################################################################
+// SIDEFRAME
 (function ($) {
     'use strict';
-    // CMS.$ will be passed for $
+
+    // shorthand for jQuery(document).ready();
     $(function () {
-        /*!
-         * Sideframe
-         * Controls a cms specific sideframe
+        /**
+         * The sideframe is triggered via API calls from the backend either
+         * through the toolbar navigation or from plugins. The APIs only allow to
+         * open a url within the sideframe
+         *
+         * @class Sideframe
+         * @namespace CMS
+         * @requires CMS.API.Helpers
```

Comment:
i think `@uses` is more appropriate
http://yui.github.io/yuidoc/syntax/index.html#uses